src/features/fwiCalculator/components/CalculationDocumentation.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Card, CardContent, Typography } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = makeStyles({
+  root: {
+    marginTop: 15,
+    marginBottom: 15,
+    width: 600
+  },
+  title: {
+    fontSize: 18,
+    marginBottom: 10,
+    color: '#000000'
+  },
+  bodyText: {
+    color: '#333333',
+    fontSize: 14
+  }
+})
+
+export const CalculationDocumentation = () => {
+  const classes = useStyles()
+
+  return (
+    <Card
+      className={classes.root}
+      id="documentation-card"
+      data-testid="documentation-card"
+    >
+      <CardContent>
+        <Typography className={classes.title}>
+          How Are These Values Calculated?
+        </Typography>
+        <Typography className={classes.bodyText}>
+          Daily weather data (collected at noon local time) for the core fire
+          seasons in the selected time range is collected for each of the
+          selected weather stations.
+          <br />
```

Comment:
Could you wrap these text with div tags with margin instead of <br /> tag? generally, <br/> is an indication of poor semantic HTML